src/redis.ts

```diff
@@ -0,0 +1,1256 @@
+    return this._commandParser(command, collection, key);
+  }
+
+  smismember(key: string, members: string[], collection: string) {
+    const command: string = "SMISMEMBER";
+    return this._commandParser(command, collection, key, ...members);
+  }
+
+  smove(
+    source: string,
+    destination: string,
+    member: string,
+    collection: string
+  ) {
+    const command: string = "SMOVE";
+    return this._commandParser(
+      command,
+      collection,
+      source,
+      destination,
+      member
+    );
+  }
+
+  spop(key: string, count: number, collection: string) {
+    const command: string = "SPOP";
+    return this._commandParser(command, collection, key, count);
+  }
+
+  srandmember(key: string, collection: string, count?: number | undefined) {
+    const command: string = "SRANDMEMBER";
+    return this._commandParser(command, collection, key, count);
+  }
+
+  srem(key: string, members: string[], collection: string) {
+    const command: string = "SREM";
+    return this._commandParser(command, collection, key, ...members);
+  }
+
+  sscan(
+    key: string,
+    cursor: number,
+    collection: string,
+    pattern?: string | undefined,
+    count?: number | undefined
+  ) {
+    const command: string = "SSCAN";
+
+    let patternArray: any[] = [];
+    if (pattern !== undefined) {
+      patternArray.push("MATCH");
+      patternArray.push(pattern);
+    }
+
+    let countArray: any[] = [];
+    if (count !== undefined) {
+      countArray.push("COUNT");
+      countArray.push(count);
+    }
+
+    return this._commandParser(
+      command,
+      collection,
+      key,
+      cursor,
+      ...patternArray,
+      ...countArray
+    );
+  }
+
+  sunion(keys: string[], collection: string) {
+    const command: string = "SUNION";
+    return this._commandParser(command, collection, ...keys);
+  }
+
+  sunionstore(destination: string, keys: string[], collection: string) {
+    const command: string = "SUNIONSTORE";
+    return this._commandParser(command, collection, destination, ...keys);
+  }
+  // End of SET commands
+
+  // Start of SORTED SET commands
+  zadd(key: string, data: any[], collection: string, options: any[] = []) {
+    const command: string = "ZADD";
+    return this._commandParser(command, collection, key, ...options, ...data);
+  }
+
+  zcard(key: string, collection: string) {
+    const command: string = "ZCARD";
+    return this._commandParser(command, collection, key);
+  }
+
+  zcount(key: string, minimum: string, maximum: string, collection: string) {
+    const command: string = "ZCOUNT";
+    return this._commandParser(command, collection, key, minimum, maximum);
+  }
+
+  zdiff(
+    numKeys: number,
+    keys: string[],
+    collection: string,
+    withScores: boolean = false
+  ) {
+    const command: string = "ZDIFF";
+    let withScoresCommand: string | undefined;
+    if (withScores) {
+      withScoresCommand = "WITHSCORES";
+    } else {
+      withScoresCommand = undefined;
```

Comment:
you can directly assign the default value at line no 612 and have only an if condition. check for all similar instances.

Reply:
done 👍